### DISCUSSION TITLE
Academic Verification Rejected While In Campus

### DISCUSSION THREAD

**Discussion by ibraheemalayan** [community/community#51917] [GitHub Education] [answered]
This the 4th request I submit, and it keeps being rejected saying:

'''

The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

'''

Although I submitted from the campus while connected to the school edurom network, and used my phone camera with access to location.

Any ideas how to solve this ?

**Comment by samus-aran** [comment]
Hi @ibraheemalayan 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by Huzaifa1910** [reply]
Recently, my GitHub developer student account requested me to re-verify my document, and despite trying every possible step mentioned in the articles, whenever I try to submit it, an error message appears saying that GitHub is unable to verify my document. I have tried multiple times, but unfortunately, the same error message appears repeatedly.

I understand that document verification is a crucial step to ensure the security and authenticity of user accounts. Therefore, I am reaching out to request your assistance in resolving this issue. I kindly ask that you provide me with guidance on how to resolve this error so that I can complete the verification process.

**Comment by ibraheemalayan** [comment]
@samus-aran 
thanks for the response, are there any updates ?

where does the school campus location/country data come from ?

**Reply by samus-aran** [reply, marked as answer]
Hi @ibraheemalayan 

We have updated your selected school and processed your application 🎉 Let us know you've received the application notification 👍

**Reply by joshcannotcode** [reply]
Hi @samus-aran, I’m having the same issue and have tried a number of different submissions but none of them have worked - would you mind looking into it for me? Thanks :)

**Reply by Sazid99246** [reply]
Hi, @samus-aran , I am having different problems. My university gave me an email, but when going to apply, it says, we require applicants to use school-issued email. I have 2 emails in my account. The email I was going to apply with, was my university issued email. Please take a look at this and let me know if I can get it or not.

**Comment by tmnrkmb** [comment]
I'm also having the same problem. I'm afraid the ID card of our University is in Japanese.

**Comment by BigTree0709** [comment]
I have same problem. I do not know what should I do. can someone help me?
